evap/results/views.py

```diff
@@ -68,14 +68,12 @@ def course_detail(request, semester_id, course_id):
         public_view = request.GET.get('public_view') == 'true'  # if parameter is not given, show own view.
 
     # If grades are not published, there is no public view
-    if not course.has_enough_voters_to_publish_grades:
+    if not course.has_visible_results:
         public_view = False
 
     represented_users = list(request.user.represented_users.all())
     represented_users.append(request.user)
 
-    show_grades = course.can_user_see_grades(request.user)
-
     # remove text answers and grades if the user may not see them
```

Comment:
which filtering do you mean?

Reply:
...if i understand correctly, the filtering got moved into `calculate_results` since it's now the same for all users, that's why it got removed here.

whether the filtering is necessary at all or not depends on the answer to my other question, why we need to apply the VOTER_COUNT_NEEDED_FOR_PUBLISHING per question and not only to the whole course.

Reply:
yes. this is all part of "publishing per question" vs. "publishing per course". as we now decided to keep our current approach and either publish everything when we have VOTER_COUNT_NEEDED_FOR_PUBLISHING voters, this will all be changed again.

Reply:
many other comments address the same thing, i will not comment individual answers and instead just push a new version when i addressed all comments. the new version will differ in quite some parts.